content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -575,28 +575,29 @@ We have a lot of features in our roadmap. An abbreviated high level list is incl
 
 {{< feature-state for_k8s_version="v1.18" state="alpha" >}}
 
-{{< glossary_tooltip term_id="containerd" >}} is another OCI-compliant runtime that works with Kubernetes on Linux. Kubernetes v1.18 is the first version tested with ContainerD on Windows. It's still a work-in-progress, and you can track progress in [enhancements#1001](https://github.com/kubernetes/enhancements/issues/1001). 
+{{< glossary_tooltip term_id="containerd" >}} is an OCI-compliant runtime that works with Kubernetes on Linux. Kubernetes v1.18 adds support for ContainerD on Windows. Progress for ContainerD on Windows can be tracked at [enhancements#1001](https://github.com/kubernetes/enhancements/issues/1001).
```

Comment:
@PatrickLang @marosset, Some general input:

Is there a good reason to place this content in the `What's next` section?
Why not introduce a `ContainerD for Windows` heading (level2)?
OR
There is already a sub section on the page,
https://deploy-preview-19208--kubernetes-io-vnext-staging.netlify.com/docs/setup/production-environment/windows/intro-windows-in-kubernetes/#container-runtime

Line 578, If the name of this feature is, `ContainerD`, then I suggest starting the sentence with that name, not the tooltip. You could use the tooltip in another place in the first sentence. Also, add the `text` attribute to the `glossary_tooltip` shortcode such as: text="ContainerD".

Line 580, The `caution` shortcode should contain the list of isssues/things.
Line 584. Perhaps modify statement about official releases indicating that `ContainerD does not officially support Windows`.
Line 594, Do you need the word, `new`
Look through the content and try to remove content that refers to `possible future use or deprecations, etc.`.
Such as,
`Windows node support in kubeadm will come in a future release.`

Reply:
I'll move the ContainerD info under the Container Runtimes - good idea!

For the containerD support - the next release of containerD will support windows. 
Because of this I'd prefer not to say there is no official support but am also trying to avoid future looking statements.

I'll address some of the other feedback as well.